bricks/template/__brick__/{{project_name.snakeCase()}}/.github/workflows/ios_deploy_staging_to_firebase.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           ENV: ${{#mustacheCase}}secrets.ENV{{/mustacheCase}}
         run: |
           echo -e "$ENV" > .env.staging
+          echo "RELEASE_NOTE_CONTENT="$((git log -1 --merges | grep "\[") | grep . && echo "" || echo $(git log -1 --merges --format=%B))""
```

Comment:
same, the script generates `RELEASE_NOTE_CONTENT`, but iOS Fastlane reads `RELEASE_NOTE` 😵‍💫